test/compress/collapse_vars.js

```diff
@@ -894,7 +894,8 @@ collapse_vars_unary: {
     }
     expect: {
         function f0(o, p) {
-            delete o[p];
+            var x = o[p];
+            delete x;
```

Comment:
Fairly certain this is not supposed to happen - the original code wouldn't delete a field with name `p` from object `o`.